addons/point_of_sale/static/src/app/services/pos_store.js

```diff
@@ -139,6 +139,7 @@ export class PosStore extends WithLazyGetterTrap {
         this.scaleTare = 0;
         this.totalPriceOnScale = 0;
 
+        this.isSystemUser = await user.hasGroup("base.group_system");
```

Comment:
```suggestion
        this.isProductCreation = await user.hasGroup("product.group_product_manager");
```

I would like to use this product creation group for product's operations.